lessons/projects/asteroids/index.html

```diff
@@ -274,29 +299,33 @@
                             teď jen zkopíruj:
 <pre>
 def distance(a, b, wrap_size):
+    """Distance in one dirextion (x or y)"""
     result = abs(a - b)
     if result > wrap_size / 2:
         result = wrap_size - result
     return result
 
 def overlaps(a, b):
+    """Returns true iff two space objects overlap"""
```

Comment:
Both of these are intentional, though subtle.
[iff](https://en.wikipedia.org/wiki/Iff) stands for “if and only if”.
“true” stands for any value for which `bool(value) is True`.